etcdmain/main.go

```diff
@@ -26,7 +27,13 @@ func Main() {
 	checkSupportArch()
 
 	if len(os.Args) > 1 {
-		switch os.Args[1] {
+		cmd := os.Args[1]
+		if covArgs := os.Getenv("ETCDCOV_ARGS"); len(covArgs) > 0 {
+			args := strings.Split(os.Getenv("ETCDCOV_ARGS"), "\xe7\xcd")[1:]
```

Comment:
string separator, in case there's a space in the arguments; etcdctl has something similar since the arguments can't be passed via command line when enabling coverage instrumentation